src/containers/Sidebar/SidebarBottomItem.js

```diff
@@ -51,11 +51,31 @@ const SidebarBottomItem = ({ item }) => {
 								title={t(`Sidebar|${item.name}`)}
 								className={`sidebar-item-button${location.pathname === item.url ? " active " : " "}btn left`}
 							>
-								<Icon icon={item.icon} />
-								<div className="sidebar-item-name ml-2">{t(`Sidebar|${item.name}`)}</div>
-								{item.children && item.children.length > 0 && (
-									<Icon icon={isOpen ? "cil-arrow-circle-bottom" : "cil-arrow-circle-left"} />
-								)}
+								{sidebarLogo ? 
+									isSidebarCollapsed ? 
+										<img
+											src={sidebarLogo?.minimized}
+											alt={t(`About`)}
```

Comment:
@petrKavulok And here, I think at the end, it should be alt={t(`Sidebar|${item.name}`)}